packages/create-plugin/templates/backend-app/pkg/plugin/app_test.go

```diff
@@ -0,0 +1,135 @@
+package plugin
```

Comment:
Yeah you are correct. Originally I didn't have resources.go and had everything in app.go, so that's why the tests are in app_test.go. I will rename it :smile: